src/components/CohortBuilder/Summary/index.js

```diff
@@ -92,6 +94,7 @@ const Summary = ({ theme, sqon, api }) => (
         <Row nogutter> no data</Row>
       ) : (
         <Row nogutter>
+          {!sqon ? <EmptyCohortOverlay /> : null}
```

Comment:
Under what condition would a falsey `sqon` gets here? In case of falsey sqon though, we should be showing the full dataset rather than `EmptyCohortOverlay`, but I don't think we are expecting a falsey sqon.

I think the right check here would be to use the `cohortResults` data, either passed in from above or reused in the `QueriesResolver` here.

Reply:
Easiest way to implement the "overlay" - if the overlay query on participants is empty, just pass an empty sqon to match this value on the other side.